changelog.md

```diff
@@ -0,0 +1,15 @@
+# Change Log
+This file contains all the notable changes done to the Ballerina Task package through the releases.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [2.0.0-alpha6] - 2021-04-02
```

Comment:
What happened to alpha5 ? :)

Reply:
I have changed to alpha5 as This change was added in the alpha5.